### DISCUSSION TITLE
Can't create new PR for mariadb@10.5 (for 10.5.17)

### DISCUSSION THREAD

**Discussion by shanecav84** [Homebrew/discussions#3806] [Writing Formulae/Casks] [answered]
An updated formula is here https://github.com/Homebrew/homebrew-core/pull/108131

That PR has been closed due to inactivity. When I run `brew bump mariadb@10.5 --open-pr` to open a new PR for 10.5.17, then I get the error message:

> Error: These pull requests may be duplicates:
> mariadb@10.5 10.5.17 https://github.com/Homebrew/homebrew-core/pull/108131
> Duplicate PRs should not be opened. Use --force to override this error.

But using `--force` gives me the error `Error: invalid option: --force`.

How can I open a new PR for this upgrade?

**Comment by SMillerDev** [comment, marked as answer]
Use `brew bump-formula-pr`
